### DISCUSSION TITLE
Bring back the visual difference between sig-prename and sig-name

### DISCUSSION THREAD

**Discussion by viktorasm** [pradyunsg/furo#311] [Potential Issue] [answered]
If you check latest docs for `urllib3`, you can see that class module and class name is rendered differently, e.g.:

https://urllib3.readthedocs.io/en/stable/reference/urllib3.util.html#urllib3.util.SSLContext

When trying this with latest version `2021.11.16`, both package and classname is rendered visually the same. I guess the previous behavior was the intended behavior, and it would be nice to retain this. For now reverting my builds to version `2021.09.22` (as used by `urllib3` at the time of this post)

**Comment by viktorasm** [comment]
Did more testing and it seems like this got introduced in 2021.11.15. Checking the diff with the previous release shows that there were changes on `.sig` styling:

https://github.com/pradyunsg/furo/compare/2021.11.12.1...2021.11.15

**Reply by viktorasm** [reply]
as a hack, added a simple override directly in documentation. Tried various options to differentiate from the name, going with bold+param color seemed to look best:

```css
.sig .sig-prename {
    color:  rgb(90, 92, 99);
}
```

![image](https://user-images.githubusercontent.com/2631366/142991647-d2d5d583-c913-4b8a-b8f8-dd13d91d2fdd.png)

**Comment by pradyunsg** [comment, marked as answer]
Done in 782574e60d4e6383b455720a06e57f8f4eea5087
